org.adempiere.base.callout/src/org/adempiere/base/callout/CostAdjustmentLineProduct.java

```diff
@@ -61,7 +61,10 @@ public String start(Properties ctx, int WindowNo, GridTab mTab,
 				trxName = gt.get_TrxName();
 			}
 		}
-		MInventory inventory = new MInventory(ctx, (Integer) mTab.getValue("M_Inventory_ID"), trxName);
+		int invId = 0;
+		if (mTab.getValue("M_Inventory_ID") != null)
+			invId = (Integer) mTab.getValue("M_Inventory_ID");
```

Comment:
should immediate return null if mTab.getValue("M_Inventory_ID") is null

Reply:
Thanks @hengsin - committed your suggestion